sense-android-library-test/src/nl/sense_os/datastorageengine/test/TestDataStorageEngine.java

```diff
@@ -271,7 +280,8 @@ public void testPeriodicSync () throws SensorProfileException, SchemaException,
             SensorOptions sensorOptions = new SensorOptions();
             sensorOptions.setUploadEnabled(true);
             sensorOptions.setDownloadEnabled(true);
-            Sensor noise = dataStorageEngine.createSensor(source, sensor_name, sensorOptions);
+            Sensor noise = dataStorageEngine.getSensor(source, sensor_name);
+            noise.setOptions(sensorOptions);
```

Comment:
@tedschmidt i think the default settings for noise is already uploadEnabled and downloadEnabled. So this will change nothing i guess?